packages/shared/components/popups/CreateAccount.svelte

```diff
@@ -94,9 +94,7 @@
                 balance={'0 Mi'}
```

Comment:
Creating an account doesn't work properly for me. I did the following:

1) Click create an account
2) Give name and color
3) Click 'Create'
4) Click 'Cancel' while waiting

After this, the wallet doesn't show in the dropdown. It might be unrelated to this PR, so feel free to open a task about it and merge this. I just wanted to let you know :smile: